src/panels/calendar/ha-full-calendar.ts

```diff
@@ -34,33 +39,43 @@ declare global {
   }
 }
 
-const fullCalendarConfig = {
+const defaultFullCalendarConfig = {
   headerToolbar: false,
-  plugins: [dayGridPlugin],
+  plugins: [dayGridPlugin, listPlugin, interactionPlugin],
```

Comment:
I would like to only load the code for the plugins we need based on the options given to the Calendar Element. Not sure the best way to do that as well as if I should put that in a new PR

Reply:
I have an Idea for this. Using the same methods for edit code in the view but would rather save it for the PR after this one if possible.